### ISSUE TITLE
latest commit in relx breaks dependencies

### ISSUE THREAD

**Issue by d-b** [bitwalker/exrm#136]
```
db@vmbox ~/elixir/exrm/test/fixtures/example_app $ mix deps.get
* Getting relx (git://github.com/erlware/relx.git)
Cloning into '/home/db/elixir/exrm/test/fixtures/example_app/deps/relx'...
remote: Counting objects: 2751, done.
remote: Compressing objects: 100% (24/24), done.
remote: Total 2751 (delta 8), reused 0 (delta 0), pack-reused 2727
Receiving objects: 100% (2751/2751), 4.12 MiB | 1.68 MiB/s, done.
Resolving deltas: 100% (1682/1682), done.
Checking connectivity... done.
** (Mix) :path option can only be used with mix projects, invalid path dependency for :erlware_commons
db@vmbox ~/elixir/exrm/test/fixtures/example_app $
```

**Comment by sheharyarn** [comment]
Same problem here. :(

**Comment by sheharyarn** [comment]
Same issue here, using this as a temporary measure:

```elixir
defp deps do
  [
    # ...
    {:relx, github: "erlware/relx", tag: "v2.0.0"}
  ]
end
```

**Comment by bitwalker** [comment]
This appears to be fixed in relx master at the moment, can you confirm on your end?

**Comment by sheharyarn** [comment]
@bitwalker Nope, still get the same error. I even specified the `master` branch explicitly. I've tracked down the commit where the issue began.

It works fine up to the commit [`4f38907`](https://github.com/erlware/relx/commit/4f38907d4967d672d3a2b13422cde2e13a42a604). The issue occurs after commit [`9ad84ac`](https://github.com/erlware/relx/commit/9ad84ac8f93756610fc99fe6a8eb3412218478aa) was made on [`erlware / relx`](https://github.com/erlware/relx).

cc: @tsloughter

**Comment by bitwalker** [comment]
Yeah Tristan and I are working on it right now, should have a fix pushed before long.

**Comment by JonahBraun** [comment]
New user here, it might be a good idea to keep tag: "v2.0.0" (or whatever latest version is working) to keep this from happening again. It was rather disappointing as a new user to follow the exrm readme instructions and get a confusing error.

@sheharyarn thx for the fix.

**Comment by bitwalker** [comment]
@JonahBraun Yeah the issue cropped up because relx has made itself available on hex.pm, and due to the changes to accomodate that, it introduced a problem on our end since I was still using the git ref. Once I move to the hex dependency, it will be locked to a specific version.

**Comment by tsloughter** [comment]
Published relx 2.1.1, please try it.

**Comment by bitwalker** [comment]
@tsloughter Thanks for the quick turnaround, everything looks good now!

This fix has been pushed in a new release, 0.17.1

**Comment by sheharyarn** [comment]
:+1:
